README.md

```diff
@@ -160,19 +137,19 @@ ALTER SESSION SET SCRIPT_LANGUAGES='<LANGUAGE_ALIAS>=localzmq+protobuf:///<bucke
 
 ## Limitations
 
-* Caution with symbolic links: 
+* Caution with symbolic links:
   If you use symbolic links inside any directory of the command line arguments
-  they must not point to files or directories outside the root of the path of the 
+  they must not point to files or directories outside the root of the path of the
```

Comment:
That was I think only an issue with starter scripts, but in general a good advice

Reply:
I removed this and the following section